src/status_im/ui/components/qr_code/styles.cljs

```diff
@@ -0,0 +1,97 @@
+(ns status-im.ui.components.qr-code.styles
+  (:require [status-im.ui.components.colors :as colors])
+  (:require-macros [status-im.utils.styles :refer [defstyle]]))
+
+(def qr-code-hint
+  {:color          colors/gray
+   :padding-bottom 24
+   :text-align     :center})
+
+(def qr-code-padding
+  15)
+
+(defn qr-code-container [dimensions]
+  {:background-color colors/white
+   :width            (:width dimensions)
+   :align-items      :center
+   :justify-content  :center
+   :padding          qr-code-padding
+   :border-radius    8})
+
+(def toolbar-done-text-ios
+  {:color          colors/blue
+   :letter-spacing -0.2
+   :font-size      17})
```

Comment:
Those can most probably be constant.

Reply:
If I do that, we'll end up with constants like `font-size-12`, `font-size-17`. In order to do this, we need a font size palette as well with names like `normal`, `small`, `large`, `title`, etc.